src/main.rs

```diff
@@ -67,7 +71,20 @@ fn validate_simulation() {
 
 /// Simulate the rabbit population
 fn simulate(starting_rabbits: i128) -> i32 {
-    todo!()
+    let mut count = 1;
+    let mut rabbits = starting_rabbits;
+    loop {
+        if rabbits % 2 == 0 {
+            rabbits = rabbits / 2;
+        } else {
+            rabbits = 3 * (rabbits) + 1;
```

Comment:
I made a mistake in the readme, so it should have been `(3 * rabbits) + 1`, but that could even have the parenthesis removed. In this case, we can remove the parenthesis around `(rabbits)` :)